notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>Again: here I would note that there is a large degree of variance in this data. I would note here about how that means that we may need more records to draw statistically significant conclusions, but we can still look at the data to see what might be happening. If we see patterns across multiple datasets and multiple species, then we might find significant results in spite of the issues of sampling bias and undersampling.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>That is true. We could not acquire significant amount of data to arrive at a definite conclusion. During my experiment I could not find any species that has got a good amount of data among 29 species I queried about. Even for turtle species that are highly reported as compared to many other species, the variance was very large.</p>